content/en/news/releases/1.16.x/announcing-1.16/change-notes/index.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Istio 1.16.0 Change Notes
+linktitle: 1.16.0
+subtitle: Minor Release
+description: Istio 1.16.0 change notes.
+publishdate: 2022-11-15
+release: 1.16.0
+weight: 10
+---
+
+## Deprecation Notices
+
+These notices describe functionality that will be removed in a future release according to [Istio's deprecation policy](/docs/releases/feature-stages/#feature-phase-definitions). Please consider upgrading your environment to remove the deprecated functionality.
+
+- **Deprecated** fetching charts from URLs in `istio-operator`.
+
+## Traffic Management
+
+- **Improved** sidecar `Host` header matching to ignore port numbers by default. This can be controlled by the `SIDECAR_IGNORE_PORT_IN_HOST_MATCH` environment variable. ([Issue #36627](https://github.com/istio/istio/issues/36627))
+
+- **Updated** `meshConfig.discoverySelectors` to dynamically restrict the set of namespaces where istiod creates the `istio-ca-root-cert` configmap
+  if the `ENABLE_ENHANCED_RESOURCE_SCOPING` feature flag is enabled.
+
+- **Updated** `meshConfig.discoverySelectors` to dynamically restrict the set of namespaces where istiod discovers Custom Resource configurations
+  (like Gateway, VirtualService, DestinationRule, Ingress, etc.) if the `ENABLE_ENHANCED_RESOURCE_SCOPING` feature flag is enabled.
+  ([Issue #36627](https://github.com/istio/istio/issues/36627))
+
+- **Updated** the gateway-api integration to read `v1beta1` resources for `HTTPRoute`, `Gateway`, and `GatewayClass`. Users of the gateway-api must
+  be on version 0.5.0+ before upgrading Istio.
+
+- **Added** support for MAGLEV load balancing algorithm for consistent hashing.
+
+- **Added** the creation of inbound listeners for service ports and sidecar
+  and ingress listener both using environment variable
+  `PILOT_ALLOW_SIDECAR_SERVICE_INBOUND_LISTENER_MERGE`.
+  Using this, the traffic for a service port is not sent via passthrough TCP even
+  though it is regular HTTP traffic when sidecar ingress listener is defined.
+  In case the same port number is defined in both sidecar ingress and service,
+  sidecar always takes precedence.
+  ([Issue #40919](https://github.com/istio/istio/issues/40919))
+
+- **Added** support for restricting istiod `WebhookCertPatcher` to only patch the webhooks it owns
```

Comment:
can you remove this? Thinking of reverting this PR, as this has some issues which are not yet sorted out.. Better not to add in release note